tests/integration/ceph_base_keystone_test.go

```diff
@@ -982,6 +982,8 @@ func runSwiftE2ETest(t *testing.T, helper *clients.TestClient, k8sh *utils.K8sHe
 }
 
 func testInOpenStackClient(t *testing.T, sh *utils.K8sHelper, namespace string, projectname string, username string, expectNoError bool, command ...string) {
+	err := sh.WaitForLabeledPodsToRun("app=osc-admin-admin", namespace)
```

Comment:
This is independent from the ceph version change, right? How about a separate PR to fix this test?

Reply:
created separate PR #15431